modules/bmo-troubleshooting-latency-when-deleting-a-baremetalhost-resource.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ipi-install-troubleshooothing-latency-when-deleting-a-baremetalhost-resource_{context}"]
+
+= Troubleshooting latency when deleting a BareMetalHost resource
+
+When the Bare Metal Operator (BMO) deletes a `BareMetalHost` resource, Ironic deprovisions the bare-metal host. For example, this might happen when scaling down a machine set. Deprovisioning involves a process known as "cleaning". If cleaning doesn't succeed, the deletion of the `BareMetalHost` resource will take a long time, because it retries three times after a timeout. The deletion might not finish.
+
+[IMPORTANT]
+====
+Do not remove finalizers from `BareMetalHost` resources that take a long time to get deleted.
```

Comment:
```suggestion
Do not remove finalizers from `BareMetalHost` resources before deletion.
```
Again, not sure if this says the same thing. I'm trying to keep the note as clear as possible.

Reply:
That's slightly misleading. So I just removed the "before deletion" part.